python/pyspark/pandas/series.py

```diff
@@ -976,9 +976,10 @@ def cov(self, other: "Series", min_periods: Optional[int] = None) -> float:
         else:
             return sdf.select(F.covar_samp(*sdf.columns)).head(1)[0][0]
 
-    # TODO: arg should support Series
     # TODO: NaN and None
```

Comment:
I guess this `TODO` can also be removed

Reply:
We'd better keep it for empty dict `arg` because of its inconsistency with pandas. I made the TODO more clear.